pkg/db/src/implementations/version-1.1/note.ts

```diff
@@ -237,11 +232,7 @@ class Note implements NoteInterface {
 				};
 			};
 
-			if (isTuple(params)) {
-				updateQuantity(params[0], params[1], params[2]);
-			} else {
-				params.forEach((update) => updateQuantity(update[0], update[1], update[2]));
-			}
+			params.forEach((update) => updateQuantity(update.isbn, update.quantity, versionId(update.warehouseId || this.#w._id)));
```

Comment:
There's one problem with falling back to `this.#w._id` and that is: we don't want any volume transaction to be assigned to `0-all` (default warehouse)...
In terms of UX, we should allow adding of transactions with no warehouses (and updated later), only we wouldn't allow for committing of notes without warehouse.